src/blog/2022/11/flowforge-1-1-released.md

```diff
@@ -0,0 +1,62 @@
+---
+title: FlowForge 1.1 released
+subtitle: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
+description: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
+date: 2022-11-24 18:00:00.0
+authors: ["rob-marcer"]
+video: TODO
+---
+
+Edit your flows directly on your FlowForge Devices, persist files on your FlowForge projects, and import Node-RED snapshots into FlowForge.
+
+<!--more-->
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above. 
+
+We're pleased to announce version 1.1 is now available. The next release of the FlowForge application containing new features, a number of improvements, and bug fixes. Keep reading for a promotion code to get your first month free on FlowForge Cloud. 
+
+
+## Features
+[Node-RED Editor on Devices](https://github.com/flowforge/flowforge/issues/1012) We've added the ability for you to edit flows directly on your Devices. 
+
+[Persistant File Storage](https://github.com/flowforge/flowforge/issues/998) We've completed the first steps to allow files to be created and persisted within your projects. This can be used to store files locally to log data or store current settings for later retrieval. 
```

Comment:
I'd go with something more like: 

Files are important to store analysis or to start processing of data in Node-RED. Using files on FlowForge Cloud or self-managed installs on Kubernetes or Docker use containers to store files. When a stack is updated to for example upgrade the Node-RED version these files are lost. Persistant files store these files on a persistant object storage. The workflow doesn't change for the user, as we've replaced the file nodes.

That states the problem with 1.0, when a user might run into this problem, and lastly how we help them to fix it.

Reply:
Also, link to the docs please

Reply:
I've reworded the explanations for each headline feature. Agreed that links to the docs would be useful but I'll need help to work out how to link to docs which are not yet in place. I suggest once we've rolled out 1.1 we edit in the docs links to the release blog.